packages/core/drizzle/0021_fancy_scourge.sql

```diff
@@ -0,0 +1,5 @@
+DO $$ BEGIN
+ CREATE TYPE "latitude"."provider" AS ENUM('openai', 'anthropic', 'groq', 'mistal');
```

Comment:
Let's see if updating an enum is not painful

Reply:
typo no? is `mistal`?

Reply:
good catch!